emails/migrations/0062_move_profile_and_registered_subdomain_models.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.13 on 2024-06-07 22:08
+# Move Profile and RegisteredSubdomain to privaterelay app
+# Other half is
+# privaterelay/migrations/0010_move_profile_and_registered_subdomain_models.py
+# See https://davit.hashnode.dev/django-move-model
```

Comment:
(praise) Glad to see this documentation!
(suggestion, non-blocking) Add that it uses Solution # 2 of the referenced documentation that uses the model's Meta option [db_table](https://docs.djangoproject.com/en/3.0/ref/models/options/#db-table).